packages/proton-bridge/build.sh

```diff
@@ -0,0 +1,22 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/ProtonMail/proton-bridge
+TERMUX_PKG_DESCRIPTION="ProtonMail Bridge application"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_VERSION=1.4.5
+TERMUX_PKG_SKIP_SRC_EXTRACT=true
```

Comment:
You can remove TERMUX_PKG_SKIP_SRC_EXTRACT and skip overriding termux_step_get_source and instead specify `TERMUX_PKG_SRCURL=https://github.com/ProtonMail/proton-bridge.git` and `TERMUX_PKG_GIT_BRANCH="br-$TERMUX_PKG_VERSION"`
See https://github.com/termux/termux-packages/blob/a1972fa63159d417a33cdee36b4a3cd976d851a9/packages/mbedtls/build.sh#L4-L6 for another place where this is done.

Reply:
Thanks, didn't found that case.

Reply:
I pushed a new commit with this.